src/borg/archiver.py

```diff
@@ -2626,34 +2626,56 @@ def do_break_lock(self, args, repository):
             EOF
             $ borg create --exclude-from exclude.txt backup /
 
-        A more general and easier to use way to define filename matching patterns exists
-        with the ``--pattern`` and ``--patterns-from`` options. Using these, you may
-        specify the backup roots (starting points) and patterns for inclusion/exclusion.
-        A root path starts with the prefix `R`, followed by a path (a plain path, not a
-        file pattern). An include rule starts with the prefix +, an exclude rule starts
-        with the prefix -, an exclude-norecurse rule starts with !, all followed by a pattern.
+        A more general and easier to use way to define filename matching patterns
+        exists with the ``--pattern`` and ``--patterns-from`` options. Using
+        these, you may specify the backup roots, default pattern styles and
+        patterns for inclusion/exclusion.
 
-        .. note::
+        Root path prefix ``R``
+            A root path starts with the prefix `R`, followed by a path (a plain
+            path, not a file pattern). Use this prefix to move root paths from
+            command line arguments to a patterns file.
```

Comment:
maybe it should use "recursion root paths" at least in the explanation.

Reply:
I didn't find the right words. Nice!